src/diffusers/modeling_flax_utils.py

```diff
@@ -45,6 +45,7 @@ class FlaxModelMixin:
     """
     config_name = CONFIG_NAME
     _automatically_saved_args = ["_diffusers_version", "_class_name", "_name_or_path"]
+    _flax_internal_args = ["name", "parent"]
```

Comment:
@pcuenca - would that be ok for you?

Reply:
Yes, that feels much better than using something that subclasses could change. I think it's perfect! :)